hw5-1/stack.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <stdlib.h>
+
+
+typedef struct StackElement {
+    int value;
+    struct StackElement *prev;

```

Comment:
```suggestion
    struct StackElement *nextElement;
```
А после первого элемента идут следующие.